test/metabase/api/database_test.clj

```diff
@@ -1040,6 +1041,7 @@
                            :schema           (s/eq (api.table/root-collection-schema-name))
                            :description      (s/maybe s/Str)}]
                          response))
+            (is (not (contains? (set (map :display_name response)) "Card 3")))
```

Comment:
Can we assert the set that we should get instead of asserting what should not be there? I would find that more convincing because that way I wouldn't ask myself if `response` might be empty or contain just `"Card 1"`.

Reply:
Unfortunately, that breaks repl usage, as this is a global list of all saved questions so pre-existing questions are included here.

Reply:
Does that mean that I can break this test by having a card named "Card 3"?